mplc/contributivity.py

```diff
@@ -62,18 +62,17 @@ def predict(self, x):
 
 
 class Contributivity:
-    def __init__(self, name="", scenario=None):
+    def __init__(self, scenario, name=""):
         self.name = name
-        n = len(scenario.partners_list)
-        self.contributivity_scores = np.zeros(n)
-        self.scores_std = np.zeros(n)
-        self.normalized_scores = np.zeros(n)
+        self.scenario = scenario
+        nb_partners = len(self.scenario.partners_list)
+        self.contributivity_scores = np.zeros(nb_partners)
+        self.scores_std = np.zeros(nb_partners)
+        self.normalized_scores = np.zeros(nb_partners)
         self.computation_time_sec = 0.0
         self.first_charac_fct_calls_count = 0
         self.charac_fct_values = {(): 0}
-        self.increments_values = []
-        for i in range(n):
-            self.increments_values.append(dict())
+        self.increments_values = [{} for _ in self.scenario.partners_list]
```

Comment:
Sure about this? Just asking

Reply:
I just rewrite the initialization of self.increments_values. I don't know where it is use.

Reply:
It is used in the importance sampling estimation of the shapley values. If I remember well, I had a pointer issue when initializing the increment values as you did. I think it is better to keep it as was.

Reply:
If you initialized this by doing [{}] * n, yes indeed you would have pointer issues. 
But with this change, it is exactly the same as before, without issue (tested)